### PR TITLE
Send registration confirmations to OpenHIM, not directly to workflow

### PR DESCRIPTION
**Pull request by rikukissa**

### CODE REVIEW

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]
https://github.com/opencrvs/opencrvs-farajaland/blob/05fa59cc6f5aada3be9893837a0186ace08558cf/src/constants.ts#L40 This also needs to be changed.

File: src/constants.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const CHECK_INVALID_TOKEN = process.env.CHECK_INVALID_TOKEN || 'false'
 export const VALIDATE_IN_BDRIS2 = process.env.VALIDATE_IN_BDRIS2 || 'false'
 export const CONFIRM_REGISTRATION_URL =
   process.env.CONFIRM_REGISTRATION_URL ||
-  'http://localhost:5050/confirm/registration'
+  'http://openhim-core:5001/confirm/registration'
```
Comment (Zangetsu101) [inline src/constants.ts:40]:
```suggestion
  'http://localhost:5001/confirm/registration'
```
I reckon we need it to be `localhost` for dev

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

**Review by euanmillar — CHANGES_REQUESTED** [review]
@rikukissa  If this PR depends upon this Core PR: https://github.com/opencrvs/opencrvs-core/pull/3678 then the then the openhim-dev.gz backup must be regenerated with the new base config imported in this PR.

**Comment by euanmillar** [comment]
FYI @rikukissa I will do this as I need to regenerate the backup for application config too

**Pull request merged**